src/liballoc/collections/btree/node.rs

```diff
@@ -73,7 +73,7 @@ struct LeafNode<K, V> {
     /// This node's index into the parent node's `edges` array.
     /// `*node.parent.edges[node.parent_idx]` should be the same thing as `node`.
     /// This is only guaranteed to be initialized when `parent` is nonnull.
-    parent_idx: u16,
```

Comment:
There's a note above ('See also rust-lang/rfcs...') that could probably be removed since we're implementing the relevant changes here? Or, if not, perhaps clarified as to what more needs to be changed?

Reply:
Ah, just looked at the same code for a different reason and had the same though. ;)

Yes, I think this implements that note.